src/main/ruby/core/range.rb

```diff
@@ -326,6 +326,14 @@ def size
     end
   end
 
+  def collect
+    Truffle.primitive :range_integer_map
+
+    super
+  end
+
+  alias_method :map, :collect
```

Comment:
I like `alias_method` without vertical space so it's clear it forms a group with the `def` above.

Reply:
I just copied the style used in enumerable.rb, but easy enough to change.